src/Quasar/QuasarF/Interpreter/Internal.purs

```diff
@@ -149,17 +149,33 @@ mkRequest
   . (String → Either Error a)
   → AXF.AffjaxF RequestContent String
   → Free (Coproduct l AXFP) (Either QError a)
-mkRequest f = map (handleResult f) <<< liftF <<< right
+mkRequest f a =
+  map (map snd) $ mkRequest' f a
+
+mkRequest'
+  ∷ ∀ a l
+  . (String → Either Error a)
+  → AXF.AffjaxF RequestContent String
+  → Free (Coproduct l AXFP) (Either QError (Tuple (AX.AffjaxResponse String) a))
+mkRequest' f = map (handleResult f) <<< liftF <<< right
+
+withExpired ∷ ∀ a b. Tuple (AX.AffjaxResponse a) b → { content ∷ b, expired ∷ Boolean }
+withExpired (Tuple { headers: hs } content) =
+  { content
+  -- , expired: doSomethingWithHeaders $ Debug.spy hs
+  , expired: doSomethingWithHeaders hs
+  }
+  where
+  doSomethingWithHeaders headers = false
```

Comment:
here will need to work out what to return as headers, not 100% sure it's going to be Boolean @kRITZCREEK mentioned something to do with maybe using a timestamp so it could be used on the frontend.